app/views/coronavirus_landing_page/components/landing_page/_page_header.html.erb

```diff
@@ -7,19 +7,14 @@
     } %>
 
     <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-grid-column-full">
```

Comment:
Nitpick: we probably want to keep this at two thirds, because most of our titles are two thirds (I think for readability). However the title of this page is pretty short, so it probably wouldn't go past two thirds anyway.